src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py

```diff
@@ -0,0 +1,362 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "informatica data-management organization serverless-runtime serverless resource-by-id",
```

Comment:
What's the difference between this command and the `informatica data-management organization serverless-runtime show` command

Reply:
Good catch! Currently, they are doing the same thing. I have removed _informatica data-management organization serverless-runtime show_ command for now.